### ISSUE TITLE
Request failed with status code 500

### ISSUE THREAD

**Issue by yashrajeshhiray** [logspace-ai/langflow#209]
Hi friends,
![image](https://user-images.githubusercontent.com/30078667/234811956-379fc1e7-b242-40e1-9735-5fa099aac8cb.png)

I tried running this setup for the document question answering, its giving this error
Error building node VectorStoreAgent: 1 validation error for VectorStoreToolkit llm Can't instantiate abstract class BaseLLM with abstract methods _agenerate, _generate, _llm_type (type=type_error)

also I am not able to integrate prompt template in this setup.

**Comment by ATL64** [comment]
Did you try with OpenAI instead of ChatOpenAI? (you would be using GTP3 instead of GPT4).  I had to to that to make it work.  I think there's issues with the chat version of OpenAI, which are now fixed in langchain, but langflow uses month old version of langchain where issues are still not resolved.  I'm not 100% sure of this though, but was waiting for future releases where langchain version is bumped up to see if that fixes it.
